prboom2/src/gl_main.c

```diff
@@ -127,6 +127,9 @@ float gl_sprite_offset;       // precalcilated float value for gl_sprite_offset_
 int gl_sprite_blend;  // e6y: smooth sprite edges
 int gl_mask_sprite_threshold;
 float gl_mask_sprite_threshold_f;
+spritefuzzmode_t gl_thingspritefuzzmode;
+spritefuzzmode_t gl_weaponspritefuzzmode;
+const char *gl_spritefuzzmodes[] = {"darken", "shadow", "transparent"};
```

Comment:
I think we really only need the array of setting names in m_menu.c?

Reply:
True, I just copied what was done above, trying to follow the convention...if there is such a thing. :)

Line 121 above has the names there. Is it preferable to move these new settings directly into `m_menu.c`, even though it will separate them from the others?

Reply:
I guess it's okay like this if it has been done before. I was just wondering why an array that is only used in the menu is defined elsewhere and then made available to the menu only through a header.

Reply:
Yes, I agree it probably should be refactored. Maybe a new PR to clean this up at some point, if it becomes necessary.